.github/workflows/chromatic.yml

```
@@ -1,10 +1,18 @@
 name: "Chromatic"
 
-on: push
+on:
+  workflow_dispatch:
+  pull_request:
```

```suggestion
  pull_request:
    types:
    - synchronize
    - labeled
```

I think this will fix [this issue](https://github.com/mbta/skate/pull/2368/files#r1457935969)